dist2src/worker/processor.py

```diff
@@ -145,7 +145,16 @@ def update_project(self, project: PagureProject, conversion_tag: str):
         Pushgateway().push_created_update()
 
     def cleanup(self):
-        if self.dist_git_dir:
-            shutil.rmtree(self.dist_git_dir, ignore_errors=True)
-        if self.src_git_dir:
-            shutil.rmtree(self.src_git_dir, ignore_errors=True)
+        """
+        Clean up the working directory.
+
+        This is safe, as long as no parallel work is done in this directory.
```

Comment:
Isn't this (parallel work) the case of more workers?

Reply:
Nope, as each worker has it's own volume, and workers are configured to execute one task at a time.